src/components/menu/editor-toolbar-menu.jsx

```diff
@@ -5,17 +5,19 @@ import PropTypes from 'prop-types';
 import NotebookIconMenu from './icon-menu'
 import tasks from '../../actions/task-definitions'
 import NotebookMenuItem from './notebook-menu-item'
+import { connectionModeIsServer } from '../../tools/server-tools'
 
 
 export class EditorToolbarMenuUnconnected extends React.Component {
   static propTypes = {
+    isServer: PropTypes.bool.isRequired,
     isAuthenticated: PropTypes.bool.isRequired,
   }
 
   render() {
     return (
       <NotebookIconMenu>
-        <NotebookMenuItem task={tasks.newNotebook} />
+        {this.props.isServer ? <NotebookMenuItem task={tasks.newNotebook} /> : undefined }
```

Comment:
`this.props.isServer && <NotebookMenuItem task={tasks.newNotebook} />` is a little more concise and has the same effect